pkg/clusteroperator/clusteroperator.go

```diff
@@ -145,6 +167,44 @@ func (builder *Builder) WaitUntilConditionTrue(
 		})
 }
 
+// WaitForClusteroperatorAvailability waits until specific clusteroperator will reach available state.
+func WaitForClusteroperatorAvailability(
```

Comment:
According to code I can see WaitUntilAvailable function already implemented. Can it be used instead of this new function?

You'll need just:
```
coObject, err := clusteroperator.Pull(apiClient, "OPERATOR NAme")
Expect(err).ToNot(HaveOccured())
err := coObject.WaitUntilAvailable()
Expect(err).ToNot(HaveOccured())
```

Reply:
sure